mesonbuild/mesonmain.py

```diff
@@ -33,7 +33,7 @@ def add_setup_arguments(p):
     p.add_argument('-v', '--version', action='version',
                    version=coredata.version)
     # See the mesonlib.WrapMode enum for documentation
-    p.add_argument('--wrap-mode', default=WrapMode.default,
+    p.add_argument('--wrap-mode', default=None,
```

Comment:
This also changes the `--help` output, so if you need to differentiate between `None` and `default` this needs to be done in a different way.

Reply:
No the default is not printed by --help. That's actually a good point, for builtin options we manually add '(default: %s)' to the help argument, but we don't do that for other options. But for --wrap-mode the default is pretty obvious anyway, since one of the choices is named "default".